### PR TITLE
[HTML5] - Custom Element Focus Styles

### PR DESCRIPTION
**Pull request by KDSBrowne**
adds new styles to improve the previous method of displaying/indicating a focused element.

### CODE REVIEW

**Comment by antobinary** [comment]
Implementation for #3852

File: bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss [hunk 1]
```diff
@@ -18,3 +18,6 @@ $color-link: $color-primary !default;
 $color-link-hover: darken($color-link, 15%) !default;
 
 $color-gray-label: $color-gray !default;
+
+$user-list-bg: #F3F6F9;
+$list-item-bg-hover: darken(#F3F6F9, 7%);
```
Comment (OZhurbenko) [inline bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss]: I don't see this variable used anywhere. But there is a duplicate under `imports/ui/stylesheets/variables/focus.scss`.
Am I missing something?

File: bigbluebutton-html5/imports/ui/stylesheets/variables/focus.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  
+  &,
+  &:active {
+    &:focus {
+      outline: none !important;
+
+      span:first-of-type::before {
+        position: absolute;
+        content: '';
+        top: calc(2px - 3px - 3px);
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/stylesheets/variables/focus.scss]: Why use calc? 2px-3px-3px = -4px

Comment (KDSBrowne) [reply bigbluebutton-html5/imports/ui/stylesheets/variables/focus.scss]: originally I was using it to calculate em values.

File: bigbluebutton-html5/imports/ui/components/button/styles.scss [hunk 3]
```diff
@@ -37,28 +37,15 @@ $btn-jumbo-padding: $jumbo-padding-y $jumbo-padding-x;
  * ==========
  */
 .button {
-  border: $btn-border-size solid transparent;
-  border-radius: $btn-border-radius;
+  @extend .btnFocus;
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/button/styles.scss]: Use a SASS placeholder instead of a class.

**Pull request merged**